modules/ROOT/pages/configure-api-task.adoc

```diff
@@ -29,7 +29,7 @@ Not all options are available depending on the runtime as well as the API.
 |===
 | Field Name | Description | Required | Notes
 | *API instance label* | Specifies a label for the API. | No | If you have multiple managed instances of the same API, add a label to differentiate each instance from the others.
-| *Implementation URI* | The URL to access for the proxy or the API. | Yes|  For example, you can use the URL of your API asset in Exchange. 
+| *Implementation URI* | The URL to access for the proxy or the API. | No |  For example, you can use the URL of your API asset in Exchange. 
```

Comment:
I'm not sure about this change. 

I don't have a service mesh instance to test how it is being showed now.